src/importexport/musicxml/internal/musicxml/musicxmlsupport.h

```diff
@@ -268,5 +56,7 @@ extern bool isLaissezVibrer(const SymId id);
 extern const Articulation* findLaissezVibrer(const Chord* chord);
 extern String errorStringWithLocation(int line, int col, const String& error);
 extern String checkAtEndElement(const muse::XmlStreamReader& e, const String& expName);
+extern void determineTupletFractionAndFullDuration(const Fraction duration, Fraction& fraction, Fraction& fullDuration);
+extern Fraction missingTupletDuration(const Fraction duration);
```

Comment:
An alternative option would be to move them to `musicxmltupletstate.h`, to keep tuplet-related things together. I don't know if that's better though; I'll let you decide, based on what you think is a sensible way to evolve the tuplet related stuff